src/abstract/CSBondLock.sol

```diff
@@ -42,40 +42,37 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
     bytes32 private constant CS_BOND_LOCK_STORAGE_LOCATION =
         0x78c5a36767279da056404c09083fca30cf3ea61c442cfaba6669f76a37393f00;
 
-    uint256 public immutable MIN_BOND_LOCK_RETENTION_PERIOD;
-    uint256 public immutable MAX_BOND_LOCK_RETENTION_PERIOD;
+    uint256 public immutable MIN_BOND_LOCK_PERIOD;
+    uint256 public immutable MAX_BOND_LOCK_PERIOD;
 
     event BondLockChanged(
         uint256 indexed nodeOperatorId,
         uint256 newAmount,
-        uint256 retentionUntil
+        uint256 until
     );
     event BondLockRemoved(uint256 indexed nodeOperatorId);
 
-    event BondLockRetentionPeriodChanged(uint256 retentionPeriod);
+    event BondLockPeriodChanged(uint256 period);
 
-    error InvalidBondLockRetentionPeriod();
+    error InvalidBondLockPeriod();
     error InvalidBondLockAmount();
 
-    constructor(
-        uint256 minBondLockRetentionPeriod,
-        uint256 maxBondLockRetentionPeriod
-    ) {
-        if (minBondLockRetentionPeriod > maxBondLockRetentionPeriod) {
-            revert InvalidBondLockRetentionPeriod();
+    constructor(uint256 minBondLockPeriod, uint256 maxBondLockPeriod) {
+        if (minBondLockPeriod > maxBondLockPeriod) {
+            revert InvalidBondLockPeriod();
         }
-        // retention period can not be more than type(uint64).max to avoid overflow when setting bond lock
-        if (maxBondLockRetentionPeriod > type(uint64).max) {
-            revert InvalidBondLockRetentionPeriod();
+        // period can not be more than type(uint64).max to avoid overflow when setting bond lock
+        if (maxBondLockPeriod > type(uint64).max) {
+            revert InvalidBondLockPeriod();
         }
-        MIN_BOND_LOCK_RETENTION_PERIOD = minBondLockRetentionPeriod;
-        MAX_BOND_LOCK_RETENTION_PERIOD = maxBondLockRetentionPeriod;
+        MIN_BOND_LOCK_PERIOD = minBondLockPeriod;
+        MAX_BOND_LOCK_PERIOD = maxBondLockPeriod;
     }
 
-    /// @notice Get default bond lock retention period
-    /// @return Default bond lock retention period
-    function getBondLockRetentionPeriod() external view returns (uint256) {
```

Comment:
Keep the old name also for the backward capability

Reply:
Done